modules/create_timepoint/php/create_timepoint.class.inc

```diff
@@ -25,6 +25,8 @@ use \LORIS\StudyEntities\Candidate\CandID;
  */
 class Create_Timepoint extends \NDB_Form
 {
+    public $skipTemplate = true;
```

Comment:
Unsure why but react create_timepoint somehow had old template that was getting rendered before the react loaded and was unnecessary. I deleted the form_create_timepoint.tpl and added skipTemplate.

Reply:
What does that have to do with fixing php8? If it's unrelated can you please send a different PR and not include it in this one..

Reply:
@driusan, it's unrelated. I'll submit another PR.

Reply:
@driusan see https://github.com/aces/Loris/pull/7165 for the other PR.